tool/tctl/common/acl_command.go

```diff
@@ -0,0 +1,242 @@
+/*
+Copyright 2023 Gravitational, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package common
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+	"time"
+
+	"github.com/alecthomas/kingpin/v2"
+	"github.com/gravitational/trace"
+
+	"github.com/gravitational/teleport"
+	"github.com/gravitational/teleport/api/types/accesslist"
+	"github.com/gravitational/teleport/lib/asciitable"
+	"github.com/gravitational/teleport/lib/auth"
+	"github.com/gravitational/teleport/lib/service/servicecfg"
+	"github.com/gravitational/teleport/lib/utils"
+)
+
+// ACLCommand implements the `tctl acl` family of commands.
+type ACLCommand struct {
+	format string
+
+	list        *kingpin.CmdClause
+	show        *kingpin.CmdClause
+	usersAdd    *kingpin.CmdClause
+	usersRemove *kingpin.CmdClause
+	usersList   *kingpin.CmdClause
+
+	// Used for managing a particular access list.
+	accessListName string
+
+	// Used for managing membership to an access list.
+	userName string
+	expires  string
+	reason   string
+}
+
+// Initialize allows ACLCommand to plug itself into the CLI parser
+func (c *ACLCommand) Initialize(app *kingpin.Application, _ *servicecfg.Config) {
+	acl := app.Command("acl", "Manage access lists.").Alias("access-lists")
+
+	c.list = acl.Command("list", "List cluster access lists.").Alias("ls")
```

Comment:
We are pretty inconsistent w.r.t list commands, we have:

- `tctl recordings ls` 
- `tctl requests ls`
- `tctl apps ls`
- `tctl bots ls`
- `tctl proxy ls`
- `tctl alerts ack ls`
-  ...

And only a couple that use `list` instead of `ls`:

- `tctl inventory list`
- `tctl alerts list`

Do you think we should view the latter as mistakes and try to stick to `ls`?

Reply:
Let's stick to ls.